src/api/auth/signInSignUp.ts

```diff
@@ -1,10 +1,10 @@
-import makeRequest from 'src/utils/services/makeRequest';
+import makeRequest from '../makeRequest';
 
 export const signInSignUp = async (
   email: string,
   password: string,
   signURL: string,
-  changeState: () => void,
+  manageRequestMessage: (error: string) => void,
```

Comment:
I think the names used in the type could stay consistent in all the files the type is used so as the part in the brackets is `(message: string)` in all the other files the type appears, maybe it could be like this here as well?